### PR TITLE
Added photo report

### PR DESCRIPTION
**Pull request by mutaphysis**

### CODE REVIEW

**Comment by mutaphysis** [comment]
jul-16-2018 18-40-04

File: src/components/NodeToolbar/Photos/PhotoSection.js [hunk 1]
```diff
@@ -152,12 +184,20 @@ class PhotoSection extends React.Component<Props, State> {
...
         />
-        <Lightbox images={lightBoxPhotos}
+        <Lightbox
+          images={lightBoxPhotos}
           onClose={this.closeLightbox}
           onClickPrev={this.gotoPrevious}
           onClickNext={this.gotoNext}
           currentImage={currentImageIndex}
           isOpen={this.state.isLightboxOpen}
+          imageCountSeparator={' ' + t`of` + ' '}
```
Comment (opyh) [inline src/components/NodeToolbar/Photos/PhotoSection.js]: Could you add a translator hint here?

File: src/components/PhotoUpload/ReportPhotoToolbar.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    case 'wrong-place': 
+      return t`Wrong Place`;
+    // translator: title for a report option where the photo does not match the current state of the place
+    case 'outdated': 
+      return t`Outdated`;
+    // translator: title for a report option where the photo is nsfw
+    case 'offensive': 
+      return t`Offensive or Inappropriate`;
+    // translator: title for a report option where the photo has another problem
+    case 'other':
```
Comment (opyh) [inline src/components/PhotoUpload/ReportPhotoToolbar.js]: I'm not 100% sure but I think the comments must preceed the lines with the strings for c-3po to understand them…

File: src/components/PhotoUpload/ReportPhotoToolbar.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    case 'other': 
+      return t`Other Problems`;
+    default:
+      return null;
+  }
+}
+
+function reportDescription(reportValue: ReportOptions): ?string {
+  switch (reportValue) {
+    // translator: title for a report option where the photo does not match the place
```
Comment (opyh) [inline src/components/PhotoUpload/ReportPhotoToolbar.js]: Same here

File: src/components/NodeToolbar/Photos/PhotoSection.js (hunk 1 above)
Comment (mutaphysis) [reply src/components/NodeToolbar/Photos/PhotoSection.js]: Jupp

File: src/components/PhotoUpload/ReportPhotoToolbar.js (hunk 2 above)
Comment (mutaphysis) [reply src/components/PhotoUpload/ReportPhotoToolbar.js]: Will fix!

**Pull request merged**